playbooks/roles/ambari-config/templates/vdf-HDF-part-repository-info.j2

```diff
@@ -1,16 +1,15 @@
     <os family="{{ baseurl_os_family }}">
       <package-version>{{ hdf_version|replace('-','_') }}_*</package-version>
       <repo>
-        <baseurl>{{ hdf_main_repo_url }}</baseurl>
```

Comment:
I have another work around for this to make this work in a more generic environment.